assignments/objects.js

```diff
@@ -10,6 +10,7 @@
 // 5, adaine5@samsung.com, Antonietta, F
 
```

Comment:
Amazing job here, especially on the stretch I feel like you're really grasping JS and your knowledge shows!